### DISCUSSION TITLE
ipfs intigration

### DISCUSSION THREAD

**Discussion by Darin755** [copy/v86#745] [Ideas]
Would it be possible to integrate v86 with ipfs? It seems like that would be a great way to store images in a reliable manner that would have decent speeds. 

Here is how to use it in a bit more detail:
http://docs.ipfs.tech.ipns.localhost:8080/how-to/address-ipfs-on-web

**Comment by izzqz** [comment]
It's possible using websocket proxy for networking. I think standard ipfs node should work with it as well.

https://github.com/copy/v86/blob/master/docs/networking.md

**Reply by Darin755** [reply]
I was actually using it to bypass same site restrictions and to get better performance for fetching images. I didn't consider using it for networking
